### ISSUE TITLE
Permission to create python/typing-council repository

### ISSUE THREAD

**Issue by JelleZijlstra** [python/steering-council#219]
As previously mentioned [on Discuss](https://discuss.python.org/t/pep-729-typing-governance-process/35362/9?u=jelle), I'd like to create a new `python/typing-council` repo. Similar to this repo, the new repo would exist primarily so that people can open issues asking the Typing Council to take an action (e.g., changing the typing spec, or providing a recommendation on a PEP). In addition, the repo would serve as a place to post operational details about the Council, such as the current membership and procedures.

I'm posting here because of the [policy](https://discuss.python.org/t/new-python-organization-repository-policy/17376) to get confirmation from the SC before creating a new repo in the python org. The policy calls for repos to first be created in a personal account, but that doesn't seem very useful here as the repo would have little content other than its issue tracker.

**Comment by Yhg1s** [comment]
Yes, you have permission from the SC to create a python/typing-council repo.

**Comment by JelleZijlstra** [comment]
Thanks! https://github.com/python/typing-council now exists.
